shared/setup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+set -e
+
+# setup shared env vars
+. ./shared.env
+
+GENESIS=$(($(date +%s) + 60))   # 60s till genesis
+SHANGHAI=$(($GENESIS + 108))    # 108s till shanghai
+CANCUN=$(($GENESIS + 144))      # 144s till cancun
```

Comment:
These values should correspond to the `CAPELLA_FORK_EPOCH` and `EIP4844_FORK_EPOCH` in the chain config. So they should be 128 for capella ( $CapellaForkEpoch \times SecondsPerSlot \times SlotsPerEpoch$ ) and 256 for EIP-4844.